src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1831,26 +1871,32 @@ private void visitEnumType(String symbolName, String qualifiedName, EnumType typ
       emitBreak();
     }
 
+    private Set<String> collectAllLiterals(Map<String, Node> elements) {
+      Set<String> literalTypes = new HashSet();
```

Comment:
literal initializers, not types. We emit types but when we collect them they are just initializers.

Reply:
Done.
Added javadoc too.